charts/unpoller/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+name: unpoller
+description: A Helm chart for unpoller, a unifi prometheus exporter
+type: application
+version: "2.11.2"
+appVersion: "v2.11.2"
+keywords:
+  - esp32
+  - home-assistant
+home: https://github.com/unpoller/unpoller
+icon: XXX?
+sources:
+  - https://github.com/unpoller/unpoller
+maintainers:
+  - name: XXXX?
+    email: XXXX?
```

Comment:
This is part of why this feels out of place. I don't think the repo maintainers can also maintain this with high confidence. Whatcha think Cody?

Reply:
Very happy to hear that! I'll mostly bow out of this then and only poke my nose in when I'm curious.